src/main/java/org/kairosdb/datastore/cassandra/CassandraDatastore.java

```diff
@@ -201,6 +212,15 @@ public void putDataPoint(String metricName,
                              ImmutableSortedMap<String, String> tags,
                              DataPoint dataPoint,
                              int ttl) throws DatastoreException {
+
+        Span parentSpan = tracer.scopeManager().active().span();
+        Tracer.SpanBuilder spanBuild = tracer.buildSpan("putDataPoint").withTag("Name", "putData");
```

Comment:
should tag names start from Capital (`"Name"`) or small letter (`"name"`)?

Reply:
I will change it to small case to make consistent with other tag names. Not sure if a standard is available in team currently regarding this.

Reply:
It seems like semantic conventions recommends snake_case. so ``put_data_point`` and there is no reason for duplication of tags and operation names.